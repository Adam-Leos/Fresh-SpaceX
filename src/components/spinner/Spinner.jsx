import React from 'react';
import './spinner.scss';

const Spinner = () => (
  <div className="spinner">
    <div className="spinner__circle" />
    <div className="spinner__circle spinner__circle--second" />
    <div className="spinner__circle spinner__circle--third" />
    <div className="spinner__circle spinner__circle--fourth" />
    <div className="spinner__circle spinner__circle--fifth" />
  </div>
);

export default Spinner;
