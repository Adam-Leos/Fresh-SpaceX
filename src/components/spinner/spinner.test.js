import React from 'react';
import Spinner from './Spinner.jsx';

test('Spinner is matching snapshot', () => {
  expect(shallow(<Spinner />)).toMatchSnapshot();
});
