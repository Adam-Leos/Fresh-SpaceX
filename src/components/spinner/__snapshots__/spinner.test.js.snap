// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner is matching snapshot 1`] = `
<div
  className="spinner"
>
  <div
    className="spinner__circle"
  />
  <div
    className="spinner__circle spinner__circle--second"
  />
  <div
    className="spinner__circle spinner__circle--third"
  />
  <div
    className="spinner__circle spinner__circle--fourth"
  />
  <div
    className="spinner__circle spinner__circle--fifth"
  />
</div>
`;
